src/chexus/validators.py

```diff
@@ -205,6 +205,28 @@ def validate(self, node: Dataset | Group) -> Violation | None:
             return Violation(node.name)
 
 
+class chopper_frequency_units_invalid(Validator):
+    def __init__(self) -> None:
+        super().__init__(
+            "chopper_frequency_unit_invalid",
+            "The unit of NXdisk_chopper.rotation_speed should have dimension 1/Time",
+        )
+
+    def applies_to(self, node: Dataset | Group) -> bool:
+        return (
+            isinstance(node, Group)
+            and node.attrs.get('NX_class') == 'NXdisk_chopper'
+            and 'rotation_speed' in node.children
+        )
+
+    def validate(self, node: Dataset | Group) -> Violation | None:
+        if 'units' in node.children.get('rotation_speed').attrs:
+            unit = node.children.get('rotation_speed').attrs.get('units')
+            if unit.endswith('Hz') or unit.endswith('s') and '/' in unit:
```

Comment:
These checks are too loose. E.g., these two would erroneously pass: `s*Hz`, `m/s`

Reply:
I think it make sense to bring in [pint](https://pint.readthedocs.io/en/stable/getting/index.html) as a dependency. It's likely we will want to do similar checks in other places.

Reply:
Since this is a pure analysis tool, i.e., it won't affect any actual loaders, I'm for being strict.

Maybe use a regex like this: `\w*(1\w*/\w*s|[umkM]?(Hz|hz|HZ))\w*`. Though realistically, the SI prefix for Hz is unlikely to be used. The hoppers we have cosily spin at a few Hz.

Reply:
I think the regex you sent would probably work very well in this case. But I still lean towards using a library for this since it's more likely to handle surprising edge cases correctly and it seems likely we will want to do similar unit checks in other validators anyway.

Reply:
While I in principle agree, we need to be careful with using a different parser than Scipp's because the same edge cases that you are worried about can lead to differences in the parsers. So I'd rather we use Scipp for this. Or have a clear decision against using Scipp, then you can go ahead with Pint.

@SimonHeybrock ?

Reply:
I'm all for using scipp instead of pint here. I opted for pint in this case because I got the impression that it was a priority to keep the dependencies  lightweight.

Reply:
I have suggested elsewhere that we should use Scipp to parse units, but make the tool still work without. That is, either dynamically skip the checks, or add a `--use-scipp` command line option to enable additional checks.

Reply:
I'd say make checking units the default and if `scipp` can't be found then either
  1) skip the tests that require it and inform the user what happened and what they need to do to run the full test suite, or
  2) fail and inform the user they need to use a command line option like `--no-scipp` to run the tool without unit checking.

Making it opt-in I think there's risk the option will not be discovered.

Reply:
Option 2 sounds good, I think?